src/components/NavLink/NavLink.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function NavLink() {
```

Comment:
I think it's better to change NavLink name to something else because react router dom library has component with the same name